components/Footer.js

```diff
@@ -0,0 +1,26 @@
+import AddButton from './AddButton';
+import AddForm from './AddForm';
+import { useState } from 'react';
+import styled from 'styled-components';
+
+export default function Footer({ onAddItem }) {
+  const [addActive, setAddActive] = useState(false);
+  return (
+    <StyledSection>
+      {addActive === true ? (
+        <AddForm onAddItem={onAddItem} setAddActive={setAddActive} />
```

Comment:
```suggestion
        <AddForm onAddItem={onAddItem} onClick={setAddActive} />
```
See my comment in the AddButton component :)